jpo-geojsonconverter/src/main/resources/schemas/processed-map.schema.json

```diff
@@ -384,21 +382,42 @@
         },
         "IPAddress": {
             "oneOf": [
-                { 
+                {
                     "type": "string",
                     "format": "ipv4"
                 },
-                { 
+                {
                     "type": "string",
                     "format": "ipv6"
                 }
             ]
         },
         "J2735SignalGroupID": {
             "type": "integer",
-            "minimum": 0,
+            "description": "DE_SignalGroupID",
             "maximum": 255,
-            "description": "DE_SignalGroupID"
+            "minimum": 0
+        },
+        "Position3D": {
```

Comment:
Identical to "OdePosition3D" in map.schema.json, could be referenced instead of copied here?

Reply:
Not sure why I did this tbh. I reverted it.